test/Vault.t.sol

```diff
@@ -753,7 +753,8 @@ contract VaultTest is Test, SetupContract, Utilities {
 
         uint256 liquidatorSpent = oldLiquidatorBalance - token.balanceOf(liquidator);
 
-        uint256 targetTreasuryBalance = (1400 * 10**18 * protocolGovernance.protocolParams().liquidationFeeD) / 10**9;
+        uint256 targetTreasuryBalance = (1400 * 10**18 * uint256(protocolGovernance.protocolParams().liquidationFeeD)) /
```

Comment:
JFYI, depending on the context, you could also write `1400 ether` instead of `1400 * 10**18`, or `1 gwei` instead of `10**9`. Choice is up to you.